libs/elasticsearch/tests/integration_tests/test_cache.py

```diff
@@ -0,0 +1,26 @@
+from unittest.mock import MagicMock
+
+from langchain.globals import set_llm_cache
+from langchain_core.language_models import BaseChatModel
+from langchain_core.messages import AIMessage
+from langchain_core.outputs import ChatGeneration
+
+from langchain_elasticsearch.cache import ElasticsearchCache
+
+
+def test_hit_and_miss(
```

Comment:
This only seems to test that the caching mechanism in LangChain is properly implemented. But it seems like it does not test any functionality of `ElasticsearchCache`, if I'm not missing anything.

Note that in integration tests, you can assume that there is an Elasticsearch instance running. For local development you can run
```
cd libs/elasticsearch/tests/integration_tests
docker compose up
```
and in [CI it's automatically started](https://github.com/langchain-ai/langchain-elastic/blob/3a26eddd05cfdbd2cb23bf82546b399699c248b1/.github/workflows/_integration_test.yml#L29-L41).

Reply:
Yes, we used to just test it was working in LangChain. I have refactored the integration tests, trying to work with the existing utilities and the local ES instance